microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -163,14 +168,20 @@ def run_metrics_tests(
             if subtest_timeout:
                 cmd_args = f"{cmd_args} --timeout {subtest_timeout}"
             try:
+                cmd_timeout = self.PERF_CMD_TIME_OUT
+                if self.clh_guest_vm_type == "CVM":
+                    cmd_timeout = cmd_timeout + 300
```

Comment:
Does individual test in perf metrics take more than default 900 sec (PERF_CMD_TIMEOUT) for CVM ?

Reply:
Yes it does.

Reply:
@pupacha Please check latest push